docs/src/pages/components/pickers/pickers.md

```diff
@@ -24,20 +24,49 @@ packageName: '@material-ui/lab'
 
 ⚠️ Native input controls support by browsers [isn't perfect](https://caniuse.com/#feat=input-datetime).
 
-### Date picker
+Native date (`type="date"`), time (`type="time"`) and date&time (`type="datetime-local"`) pickers.
 
-A native datepicker example with `type="date"`.
+{{"demo": "pages/components/pickers/NativePickers.js"}}
 
-{{"demo": "pages/components/pickers/DatePickers.js"}}
+## Setup
```

Comment:
The setup is specific to the "React components" section. Should we move it under it?

Reply:
done in [`2951b8c` (#27394)](https://github.com/mui-org/material-ui/pull/27394/commits/2951b8cd8f433ddd14a3aed1996ae202000c1d30)